boards/PDM/Inc/Io/Io_Efuse.h

```diff
@@ -0,0 +1,145 @@
+/**
+ * @brief Driver for interfacing with the 22XS4200 E-Fuse via SPI
+ * @Settings Efuse max SPI transfer rate: 8MHz
+ *           SPI Clock Polarity: 0 (SCK low-level idle state)
+ *           SPI Clock Phase: 2nd edge (slave samples MOSI on SCK falling edge)
+ *           Slave Select: Active Low (Must be toggled between SPI messages)
+ */
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "main.h"
+#include "App_SharedExitCode.h"
+
+struct Efuse_Context;
+
+/**
+ * Allocate and initialize an Efuse
+ * @param hspi Handle to the SPI peripheral used for the Efuse
+ * @param chip_select_port Handle to Efuse's chip-select GPIO port
```

Comment:
Not sure if handle is the best way to describe GPIO ports and pins here

Reply:
_handles_ are pointers to structures, no ?

Reply:
I think the term is loosely [defined](https://stackoverflow.com/questions/1923599/pointer-vs-handles-in-c-are-the-terms-used-to-convey-separate-things). 

Just doing off the fact that everything we refer to as handles are also called handles in ST's libraries. (SPI_HandleTypeDef, TIM_HandleTypedef, etc.)

Reply:
hmm ya I honestly went with handle here because that's what it is technically and I couldn't think of a better description. This is how it's defined in HAL drivers:


GPIOx,: | where x can be (A..H) to select the GPIO peripheral for STM32L4 family
-- | --
GPIO_Pin,: | specifies the port bit to read. This parameter can be GPIO_PIN_x where x can be (0..15).

Reply:
https://os.mbed.com/users/EricLew/code/STM32L4xx_HAL_Driver/docs/tip/group__GPIO__Exported__Functions__Group2.html

Reply:
changed this to more closely match comment for BMS